main.go

```diff
@@ -287,18 +287,6 @@ func executeCLI(cmd *cobra.Command, src *source, w io.Writer) error {
 		return err
 	}
 
-	// trim lines
-	lines := strings.Split(out, "\n")
```

Comment:
not sure why this was needed, but it was breaking some things... I removed it and see no difference otherwise 🤔